src/cli/lint/internalLints/upgrade/patches/8.2.1/patchDockerImages.ts

```diff
@@ -0,0 +1,101 @@
+import { inspect } from 'util';
+
+import fg from 'fast-glob';
+import { readFile, writeFile } from 'fs-extra';
+
+import type { PatchFunction, PatchReturnType } from '../..';
+import { log } from '../../../../../../utils/logging';
+
+const DOCKER_IMAGE_REGEX = /^(FROM\s?.*)(\s)(node|python)(:.*)/gm;
+const DOCKER_IMAGE_PLATFORM_REGEX = /^(FROM\s?.*)(--platform=[^\s]+) /gm;
```

Comment:
Hmm wait I think we only want to remove if it's a hardcoded value like arm64?

Reply:
Does this help: https://docs.docker.com/reference/build-checks/from-platform-flag-const-disallowed/